circus-train-core/src/test/java/com/hotels/bdp/circustrain/core/DiffGeneratedPartitionPredicateTest.java

```diff
@@ -15,17 +15,16 @@
  */
 package com.hotels.bdp.circustrain.core;
 
+import static com.hotels.bdp.circustrain.core.HiveEntityFactory.newFieldSchema;
```

Comment:
These are in the wrong order, it was correct before. It should be first static, then non-static and within each of those the order is
```
java
javax

org
com
com.expedia
com.hotels
```